src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -595,6 +636,13 @@ public void TestSpacingOnInterpolatedString()
             TestNormalizeExpression("$\"{3: C}\"", "$\"{3: C}\"");
         }
 
+        [Fact]
+        public void TestSpacingOnRawInterpolatedString()
+        {
+            TestNormalizeExpression("$\"\"\"{3:C}\"\"\"", "$\"\"\"{3:C}\"\"\"");
```

Comment:
Can we add a space after the format specifier in input, to show whether it's actively controlled/removed?
Also, before and after the expression (` 3 ` instead of `3` in input).

Reply:
Definitely!

Reply:
Done!